Assets/_BForBoss/_Core/Scripts/Enemies/EnemyShootingBehaviour.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using Perigon.Utility;
+using Perigon.Weapons;
+using UnityEngine;
+
+namespace BForBoss
+{
+    public class EnemyShootingBehaviour : MonoBehaviour
+    {
+        [SerializeField] private float _distanceToShootAt = 4;
+        [SerializeField] private float _rotationSpeed = 15f;
+        [SerializeField] private float _shootCountDown = 1f;
+        [SerializeField] private float _aimCountDown = 0.5f;
```

Comment:
similarly, it would be nice (for potential designers especially) to have the inspector indicate these are in deg/s , s, s for example